api/envoy/extensions/tracers/opentelemetry/samplers/v3/dynatrace_sampler.proto

```diff
@@ -28,19 +30,12 @@ message DynatraceSamplerConfig {
   // The value can be obtained from the Envoy deployment page in Dynatrace.
   int32 cluster_id = 2;
 
-  // The HTTP URI to fetch the sampler configuration (root spans per minute). For example:
-  //
-  // .. code-block:: yaml
-  //
-  //    http_uri:
-  //      uri: <tenant>.dev.dynatracelabs.com/api/v2/samplingConfiguration
-  //      cluster: dynatrace
-  //      timeout: 10s
-  //
-  config.core.v3.HttpUri http_uri = 3;
+  // [#not-implemented-hide:]
+  config.core.v3.HttpUri http_uri = 3
+      [deprecated = true, (envoy.annotations.deprecated_at_minor_version) = "3.0"];
```

Comment:
According our API policy, I think this is also a break change which is not allowed? But I am ok to hide the field but keep the implementation in the code.

Reply:
> I think this is also a break change which is not allowed?
Which change exactly do you mean? I manually added `[deprecated = true]`. I thought that was your suggestion in https://github.com/envoyproxy/envoy/pull/33034#discussion_r1535571785.

`(envoy.annotations.deprecated_at_minor_version) = "3.0"` was added automatically by `proto_format.sh`